src/panel/pages/timeline/components/TimelinePane.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+import { Pane } from "../../../components";
+import { TimelinePaneSection } from "./TimelinePaneSection";
+
+/** Pane shows additional information about a selected timeline item. */
+// TODO: update data structure
+export const TimelinePane: FC<{ event: any; sections: any }> = ({
+  // event,
+  sections
+}) => (
+  <Container>
+    <Pane.Body>
+      {sections &&
+        sections.map(section => (
```

Comment:
I didn't think about this!

Do you think we're going to have to compose sections of the pane this time?

I was hoping we could get away with always having
 - Event
 - Query / Variables
 - Response

But that might not always be the case?

Reply:
Yup agreed - I probs over-generified (is that a word? 😅). Happy to split out like this for now

Reply:
Not at all, I can see where you're coming from - had to do the same with the previous events panel.

If you think we can get away with static sections, lets go for it! 

@sofiapoh might have some thoughts on this too

Reply:
I think static for now is good, we probably don't need to make this super customisable before we come up with all the different variants we might want!